src/shashank_assignment_25/FindAlphanumericString.java

```diff
@@ -0,0 +1,26 @@
+package shashank_assignment_25;
+
+public class FindAlphanumericString {
+
+	public void isAlphanumericString(String strArray[]) {
+		boolean flag = false;
```

Comment:
this variable should be in inner for loop

Reply:
reason is, once flag updated to true then you have not init that flag value with false.